org.adempiere.ui.zk/WEB-INF/src/web/js/dragdrop/attachment/drag-drop-attachment.js

```diff
@@ -0,0 +1,199 @@
+/*******************************************************************************
+ * Copyright (C) 2017 Adaxa Pty Ltd. All Rights Reserved. This program is free
+ * software; you can redistribute it and/or modify it under the terms version 2
+ * of the GNU General Public License as published by the Free Software
+ * Foundation. This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ ******************************************************************************/
+/**
+ * Drag and Drop functionality for Attachment
+ * 
+ * @author Logilite Technologies
+ * @since October 16, 2017
+ */
+function DropToAttachFiles() {
```

Comment:
Fror new script, it is better to use namespace instead of a global name. for e.g, with idempiere namespace, see org.adempiere.ui.zk/WEB-INF/src/web/js/org/idempiere/commons/layout.js